tests/platform_tests/reboot_timing_constants.py

```diff
@@ -30,7 +30,9 @@
     "COMMON": {
         "PORT_INIT|Start": re.compile(r'.*NOTICE swss#orchagent.*initPort: Initialized port.*'),
         "PORT_READY|Start": re.compile
-        (r'.*swss#orchagent.*updatePortOperStatus.*Port Eth.*oper state set down to up.*'),
```

Comment:
The PR description says `Only Broadcom platforms have been updated here.`. But here the `common` list is updated

Reply:
Good point, I rephrased the PR description to try to make it clear that it was updated for Broadcom platforms.